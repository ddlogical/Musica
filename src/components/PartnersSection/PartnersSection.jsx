import "./PartnersSection.scss";
import { memo } from "react";

function PartnersSection() {

    const partnersLogosSrc = [
        'img/partners-logo-1.png',
        'img/partners-logo-2.png',
        'img/partners-logo-3.png',
        'img/partners-logo-4.png',
        'img/partners-logo-5.png',
        'img/partners-logo-6.png',
    ];

    return (
        <section className='partners'>

            <div className='container'>

                <div className='partners__content'>

                    <h2>Our most important publishers</h2>

                    <div className='partners__list'>
                        {partnersLogosSrc.map((logoSrc, id) => {
                            return <div key={id} className={'partners__list-item'}><img className={'partners__item-img'} src={logoSrc} alt={`PartnerLogo ${id}`} /></div>
                        })}
                    </div>

                </div>

            </div>

        </section>
    );
}


export default memo(PartnersSection);