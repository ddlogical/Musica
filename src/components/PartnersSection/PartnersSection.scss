@import '../../variables.scss';

.partners {

    &__content {
        grid-column: 1 / -1;
        grid-row: 1 / 1;
    }

    &__list {
        display: flex;
        padding: 2.9rem 1.9rem 6rem 1.3rem;
        justify-content: space-around;
        align-items: center;
    }

    &__list-item {
    
    }

    &__item-img {
        width: 100%;
        height: auto;
    }

}