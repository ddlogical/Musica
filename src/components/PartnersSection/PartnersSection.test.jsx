import { render } from "@testing-library/react";

import PartnersSection from "./";

test("PartnersSection snapshot", () => {

    const view = render(<PartnersSection />);

    expect(view).toMatchSnapshot();
})