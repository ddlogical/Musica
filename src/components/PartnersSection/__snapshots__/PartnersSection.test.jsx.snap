// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PartnersSection snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="partners"
      >
        <div
          class="container"
        >
          <div
            class="partners__content"
          >
            <h2>
              Our most important publishers
            </h2>
            <div
              class="partners__list"
            >
              <div
                class="partners__list-item"
              >
                <img
                  alt="PartnerLogo 0"
                  class="partners__item-img"
                  src="img/partners-logo-1.png"
                />
              </div>
              <div
                class="partners__list-item"
              >
                <img
                  alt="PartnerLogo 1"
                  class="partners__item-img"
                  src="img/partners-logo-2.png"
                />
              </div>
              <div
                class="partners__list-item"
              >
                <img
                  alt="PartnerLogo 2"
                  class="partners__item-img"
                  src="img/partners-logo-3.png"
                />
              </div>
              <div
                class="partners__list-item"
              >
                <img
                  alt="PartnerLogo 3"
                  class="partners__item-img"
                  src="img/partners-logo-4.png"
                />
              </div>
              <div
                class="partners__list-item"
              >
                <img
                  alt="PartnerLogo 4"
                  class="partners__item-img"
                  src="img/partners-logo-5.png"
                />
              </div>
              <div
                class="partners__list-item"
              >
                <img
                  alt="PartnerLogo 5"
                  class="partners__item-img"
                  src="img/partners-logo-6.png"
                />
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="partners"
    >
      <div
        class="container"
      >
        <div
          class="partners__content"
        >
          <h2>
            Our most important publishers
          </h2>
          <div
            class="partners__list"
          >
            <div
              class="partners__list-item"
            >
              <img
                alt="PartnerLogo 0"
                class="partners__item-img"
                src="img/partners-logo-1.png"
              />
            </div>
            <div
              class="partners__list-item"
            >
              <img
                alt="PartnerLogo 1"
                class="partners__item-img"
                src="img/partners-logo-2.png"
              />
            </div>
            <div
              class="partners__list-item"
            >
              <img
                alt="PartnerLogo 2"
                class="partners__item-img"
                src="img/partners-logo-3.png"
              />
            </div>
            <div
              class="partners__list-item"
            >
              <img
                alt="PartnerLogo 3"
                class="partners__item-img"
                src="img/partners-logo-4.png"
              />
            </div>
            <div
              class="partners__list-item"
            >
              <img
                alt="PartnerLogo 4"
                class="partners__item-img"
                src="img/partners-logo-5.png"
              />
            </div>
            <div
              class="partners__list-item"
            >
              <img
                alt="PartnerLogo 5"
                class="partners__item-img"
                src="img/partners-logo-6.png"
              />
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
